tools python/setup_env.py

```diff
@@ -0,0 +1,81 @@
+import getopt
+from posixpath import basename
+import sys
+import platform
+import os
+import wget
+import shutil
+import git 
+
+TOOLCHAIN_INSTALL_DIR="/usr/local/toolchains"
+TOOL_INSTALL_DIR="/usr/local/tools"
+
+try :
+    opts, args = getopt.getopt(sys.argv[1:], "aprs:")
+except:
+    print("Error in the getopts function")
+for o , a in opts:
+    if o == "-a" :
+        TOOLCHAIN_INSTALL_DIR = input("TOOLCHAIN_INSTALL_DIR :")
+    elif o == "-p":
+        TOOL_INSTALL_DIR=input("TOOL_INSTALL_DIR :")
+    elif o == "-r":
+        PATH=input("PATH :")
+    elif o == "-s":
+        HOME=input("Give Home Directory :")
+    else :
+        print("erreur + help function ")
+    
+DEPLOY_URL="https://gist.githubusercontent.com/phillipjohnston/bb95f19d156007f99be4c10c1efdf694/raw/f2f141e31fca0a12eb391e8251efe2ce1f9e68bd/download_and_deploy.sh"
+if(platform.release()=="Darwin"):
+    os.system(f"cat  {HOME}/.bash_profile")
+    if (os.path.isfile("c:/.bash_profile")==True):
+        exec(open('c:/.bash_profile').read())
```

Comment:
based on the bash script 
if [ "$(uname)" == "Darwin" ]; then
cat << ENDOFBLOCK >> $HOME/.bash_profile
if [ -f \$HOME/.bashrc ]; then
	source \$HOME/.bashrc
fi
first i checked if the platform system is Mac (i need to change platform.release to platform.system) then since i didnt find any equivalent bash command " cat " in python scripts i used os.system() function 
then i checked if the $HOME/.bashrc is actually a regular file and not directory 

https://tldp.org/LDP/abs/html/fto.html